app/celery/letters_pdf_tasks.py

```diff
@@ -28,6 +34,24 @@ def create_letters_pdf(self, notification_id):
         current_app.logger.info("PDF Letter {} reference {} created at {}, {} bytes".format(
             notification.id, notification.reference, notification.created_at, len(pdf_data)))
         s3.upload_letters_pdf(reference=notification.reference, crown=notification.service.crown, filedata=pdf_data)
+
+        updated_count = dao_update_notifications_by_reference(
+            references=[notification.reference],
+            update_dict={
+                "billable_units": billable_units,
+                "updated_at": datetime.utcnow()
+            }
+        )
```

Comment:
i'd just call the normal dao_update_notification method here since you've still got the notification object


```
notification.billable_units = billable_units
dao_update_notification(notification)
```

then we won't need the error logging below either